conduct.md

```diff
@@ -134,14 +134,14 @@ TODO: Write an introductory paragraph.
 
 TODO: Write an introductory paragraph.
 
-#### Acceptable Conduct
+#### Level 1 Infraction
 
-- TODO
-- TODO
-- TODO
+- Students who commit violation once the will be recorded by professor and warned orally.
 
-#### Unacceptable Conduct
+#### Level 2 Infraction
 
-- TODO
-- TODO
-- TODO
+- Students who commit violation 3 times or more in total will be required to have a meeting with professor or TLs to talk about their infractions.
```

Comment:
Related to issue #8, this meeting should probably be held with only the professor.

Reply:
In this meeting, students will also develop a plan to avoid future infractions. Could this detail be added to the end of this statement?